compute/etc/neon_collector.jsonnet

```diff
@@ -0,0 +1,346 @@
+{
+  collector_name: 'neon_collector',
+  metrics: [
+    {
+      metric_name: 'lfc_misses',
+      type: 'gauge',
+      help: 'lfc_misses',
+      key_labels: null,
+      values: [
+        'lfc_misses',
+      ],
+      query: importstr './sql_exporter/queries/lfc_misses.sql',
```

Comment:
Having a separate file for each sql query seems quite unweildy ­— especially because we need to have configuration for each item in the final jsonnet file anyways.

What's your thought process there?

(or alternatively: Why not place each metric in its own file? instead of just the query)

Reply:
I can also place each metric in it's own file if that's what you would like. See commit description for why I pulled the query files out.

Reply:
What you suggest could cut down on some repetition between the autoscaling collector and the compute collector.

Reply:
Right, I read the PR description — I'm curious to hear more about your thought process: Do you think having many files here is mostly ok? How do you weigh separating each metric vs each query into their own files?

(I'm asking this because I want to hear your thoughts first, before asking for a particular solution — I'm assuming you've thought about this more than I have :smile:)

Reply:
> Right, I read the PR description — I'm curious to hear more about your thought process: Do you think having many files here is mostly ok? How do you weigh separating each metric vs each query into their own files?

Files are cheap. I'm always one to cut down on the size of files. In this case, the value of pulling the queries out had immediate value, syntax highlighting, potential for future formatting etc. I couldn't find much value in pulling the metric objects themselves out. It _could_ make the collector files more readable since you could more easily see at a glance the metrics being collected given at that point it would just be a list of jsonnet identifiers instead of a list of objects.